packs/data/strength-of-thousands-bestiary.db/bharlen-sajor.json

```diff
@@ -706,7 +706,7 @@
                 "attackEffects": {
                     "custom": "",
                     "value": [
-                        "1d6-electricity"
+                        "1d6 electricity"
```

Comment:
Will need to just come back and fix this manually.

Reply:
Yeah, agreed. Since the custom field appears to be ignored, there isn't a good workaround for now. Just making it look prettier.